README.md

```diff
@@ -1,49 +1,143 @@
-
-¹ Supports the modpack by increasing its download count, but not with monetary benefits.
-
-<details>
-  <summary>Alternative downloads</summary>
-
-_These downloads may not fully support the modpack or the mods :(_
-* [MultiMC (auto-update)](https://wiki.download.fo/readme/multimc-auto-update): [1.16.5](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.2.3/Fabulously.Optimized.MC.1.16.5.auto-update.zip) | [1.17.1](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.2.3/Fabulously.Optimized.MC.1.17.1.auto-update.zip) | [1.18.2](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.12.2/Fabulously.Optimized.MC.1.18.2.auto-update.zip) | [1.19.4](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v4.10.0/Fabulously.Optimized.MC.1.19.4.auto-update.zip) | [1.20.2](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v5.6.5/Fabulously.Optimized.MC.1.20.2.auto-update.zip) | [1.20.4](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v5.8.0-beta.11/Fabulously.Optimized.MC.1.20.4.auto-update.zip)
-* [GitHub releases](https://github.com/Fabulously-Optimized/fabulously-optimized/releases)
-* Chat Reporting Helper resource pack: [CurseForge](https://www.curseforge.com/minecraft/texture-packs/chat-reporting-helper) | 
-[Modrinth](https://modrinth.com/resourcepack/chat-reporting-helper) |
-[Planet Minecraft](https://www.planetminecraft.com/texture-pack/chat-reporting-helper) (bundled in FO)
-* Fast Better Grass resource pack: [CurseForge](https://www.curseforge.com/minecraft/texture-packs/fast-better-grass) | [Modrinth](https://modrinth.com/resourcepack/fast-better-grass) | [Planet Minecraft](https://www.planetminecraft.com/texture-pack/fast-better-grass) (bundled in FO)
-</details>
+<div align="center">
+
+<img src="https://download.fo/icon.png" alt="FO logo" width="10%" height="10%">
+
+# Fabulously Optimized
+
+[![Official Website][img-website]][url-website]
+[![Support and Contribute][img-thanks]][url-thanks]
+[![Read the Wiki][img-wiki]][url-wiki]
+
+[![Mod list][img-mods]][file-mods]
+[![Options changed][img-options]][url-options]
+[![Resource Packs][img-rp]][url-rp]
+[![Help us Translate][img-translate]][url-translate]
+
+[![Discord][img-discord]][url-discord]
+[![GitHub][img-github]][url-github]
+[![Mastodon][img-mastodon]][url-mastodon]
+
+A simple Minecraft modpack focusing on performance
+and graphics enhancements.
+
+</div>
+
+## Download
+
+[![Installation Guide][img-install]][url-install]
+[![CurseForge][img-cf]][url-cf]
+[![Modrinth][img-modrinth]][url-modrinth]
+
+> By downloading Fabulously Optimized, you have
+> acknowledged the [disclaimers][url-terms].
+
+We recommend and support five launchers, but you can also
+download the modpack from [GitHub Releases][url-app-github].
```

Comment:
The wording implies that the GitHub Releases builds can be installed without those launchers, like the vanilla variants do, but that is not the case. Either mention GitHub Releases some way differently or not at all ("We recommend and support five launchers. All those launchers can be...").